polaris/ocean/tasks/ice_shelf_2d/ice_shelf_2d.cfg

```diff
@@ -0,0 +1,125 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# the type of vertical grid
+grid_type = uniform
+
+# Number of vertical levels
+vert_levels = 20
+
+# The minimum number of vertical levels
+min_vert_levels = 3
+
+# Depth of the bottom of the ocean
+bottom_depth = 1050.0
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = z-star
+
+# Whether to use "partial" or "full", or "None" to not alter the topography
+partial_cell_type = partial
+
+# The minimum fraction of a layer for partial cells
+min_pc_fraction = 0.1
+
+# The minimum layer thickness in m
+min_layer_thickness = 0.0
+
+# Options related to ssh adjustment steps
+[ssh_adjustment]
+
+# Number of ssh adjustment iterations
+iterations = 10
+
+# Output interval for the ssh adjustment phase in hours
+output_interval = 1.0
+
+# Run duration of each ssh adjustment phase in hours
+run_duration = 1.0
+
+# Whether to adjust land ice pressure or SSH
+adjust_variable = land_ice_pressure
+
+# Time integration scheme
+time_integrator = split_explicit
+
+# Time step in seconds as a function of resolution
+rk4_dt_per_km = 6
+
+# Time step in seconds as a function of resolution
+split_dt_per_km = 6
+
+# Time step in seconds as a function of resolution
+btr_dt_per_km = 1
```

Comment:
@cbegeman, do you think it would be useful to provide an `ssh_adjustment.cfg` in the `ice_shelf` directory in the framework with these config options (or similar ones, since I suspect these time steps are likely too small to be the defaults)?  It's also fine to just document what config options are required for any task the descends from `IceShelfTask` but I suspect that many of these defaults would be appropriate for most tasks.

Reply:
I was planning to do that, which is why I put these config options in a separate section. I'll get to this by early next week.

Reply:
Done